ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +15,24 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+size_t convertMarkTo5PointSystem(size_t markIn100Point)
+{
+    std::map<size_t, size_t> conv = { {0, 1},
+                                 {29, 1},
+                                 {30, 2},
+                                 {49, 2},
+                                 {50, 3},
+                                 {69, 3},
+                                 {70, 4},
+                                 {89, 4},
```

Comment:
it's a hack to bypass the tests, please don't use duplicated values, use intervals

Reply:
updated